### ISSUE TITLE
Provide config split recipe?

### ISSUE THREAD

**Issue by TravisCarden** [acquia/blt#3011]
The [BLT config split documentation](https://github.com/acquia/blt/blob/9.1.1/docs/config-split.md) is pretty cut and dried. At least in terms of the [environment split](https://github.com/acquia/blt/blob/9.1.1/docs/config-split.md#environment-split), there doesn't seem to be anything that couldn't be automated. Should we create a recipe for it?

**Comment by lcatlett** [comment]
@TravisCarden are you referring to the `blt recipes:config:init:splits` task itself or creating recipes enhancing the default config with common things like adding the devel module to the local split?

**Comment by TravisCarden** [comment]
Oh! Look at that: I completely overlooked the existence of the `blt recipes:config:init:splits` task. Maybe we should mention it on the documentation page.

**Comment by ba66e77** [comment]
I wonder if a unified recipes page in the docs would be helpful; not that we don't need the specific documentation for the config split recipe, but a recipe index page might help people more easily find out what recipes there are.

**Comment by TravisCarden** [comment]
I don't have an opinion on the question of a unified recipes docs page, but as to the matter of surfacing the config split recipe, in particular, I think it should be mentioned on the [config split documentation page](https://github.com/acquia/blt/blob/9.1.1/docs/config-split.md). That's where I started with my setup, and given all the manual steps there I dove right in without considering whether there might already be a command for it. I've submitted [a pull request](https://github.com/acquia/blt/pull/3020) accordingly.
